lib/hideUI.js

```diff
@@ -12,10 +12,23 @@ define(function (require, exports, module) {
 	 * for bramble to be loaded
 	 */
 	function hide() {
-		removeMainToolBar();
-		removeLeftSideToolBar();
-		removeRightSideToolBar();
-		removeTitleBar();
+		if(!(_parseParams("ui") === "1")) {
+			removeMainToolBar();
+			removeLeftSideToolBar();
+			removeRightSideToolBar();
+			removeTitleBar();
+		}
+	}
+
+	/**
+	 * This function parses the current url for brackets, and looks for Get Methods
+	 */
+	function _parseParams(sParam) {
+		var sPageURL = window.location.search.substring(1);
```

Comment:
Instead of this, you can use the Brackets API to do this, see https://github.com/humphd/brackets-browser-livedev/pull/59